apps/gbridge/PROTOCOL.md

```diff
@@ -59,6 +59,23 @@ Send a response to a notification from phone
 * n can be one of "dismiss", "dismiss all", "open", "mute", "reply"
 * id, tel and message are optional
 
+## activity data
```

Comment:
To move to https://github.com/espruino/EspruinoDocs/blob/master/info/Gadgetbridge.md once API finalized.

Reply:
Should I revert this from here?

Reply:
Nevermind :)

Reply:
Sorry - I was a bit hasty merging! I'll tweak it here - I realised we probably wanted to send movement health data too